test/imports.test.ts

```diff
@@ -664,38 +664,6 @@ describe('do_ed25519_verify', () => {
   });
 });
 
-describe('do_ed25519_batch_verify', () => {
```

Comment:
These unit tests are pointless and should be nuked because:
- They're difficult to maintain, since we need to manually code complex message formats to set them up
- They don't catch bugs - they've all been passing for weeks, despite the code being broken, because they're only testing our assumptions